### PR TITLE
Chapter 3 copy edit

### PR DESCRIPTION
**Pull request by sentience**

### CODE REVIEW

File: content/03-tea/05-breakdown-view.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ title       = "View"
...
 view : Model -> Html Msg
 view model =
     div [] [ text model ]
 ```
 
-This is a function that take a `Model` and return `Html`.
+This is a function that takes a `Model` argument and returns `Html`.
 
-Note `Html Msg`. `Msg` is a type variable. This means that this `Html` emits messages of a concrete type `Msg`. 
+If you’re wondering about the `Html Msg`. The `Msg` here is called a type variable. In this example, it indicates that the `Html` that our `view` returns  will only ever emit the messages specified in our program’s `Msg` type.  Being specific about this enables the Elm compiler to guarantee that our program handles all possible messages.
```
Comment (sporto) [inline content/03-tea/05-breakdown-view.md]: I put `type variable` initially here. But this is not correct. A type variable would `a` in be `Html a`.
`Msg` in `Html Msg` is something else. A concrete type?

Maybe let's write:

A little explanation about `Html Msg`. If you see `Html msg`, the lowercase `msg` is a type variable. Meaning that `Html` can be of any type. Like generic in other languages (e.g. Html<T>).

If you see `Html Msg` it means that this type have a concrete type. In this example, it indicates that the `Html` that our `view` returns  will only ever emit the messages specified in our program’s `Msg` type.

File: content/03-tea/06-breakdown-update.md [hunk 2]
```diff
@@ -3,16 +3,14 @@ title       = "Update"
 weight      = 6
 +++
 
-The next part is the **udpate** function:
+The next part is the `udpate` function, which is equivalent to a **reducer** in Redux:
```
Comment (sporto) [inline content/03-tea/06-breakdown-update.md]: `update`, my mistake

File: content/03-tea/05-breakdown-view.md (hunk 1 above)
Comment (sentience) [reply content/03-tea/05-breakdown-view.md]: Hmm, trying to explain the difference between `Html msg` and `Html Msg` seems overly confusing for a complete beginner audience. You'll note I removed all the generic types I could from the example, specifically to explaining this issue.

I'll see if I can write something about the `Html` type taking another type as an argument, and only mention generics to say “If you're wondering if this is the generic types you’re familiar with from Rust or Java, you’re right.”

Comment (sporto) [reply content/03-tea/05-breakdown-view.md]: Agree, this is confusing for beginners. Specially if they have never done a static typed lang.

**Pull request merged**